packages/docs/src/routes/docs/(qwik)/faq/index.mdx

```diff
@@ -269,6 +270,10 @@ Most of Qwik is written in TypeScript, a superset of JavaScript that adds option
 
 Yes! there is a growing community of Qwik developers at [Discord](https://qwik.builder.io/chat) and [GitHub](https://github.com/BuilderIO/qwik). They are making amazing contributions to the framework, building sites at scale, and helping each other. [Join us](https://qwik.builder.io/chat).
 
+## Is Qwik an "Alex Russell Approved" framework?
+
+Yes! Alex Russell ([@slightlylate](https://twitter.com/slightlylate)), known for his contributions to PWAs, W3C TAG, WC, TC39 & ES6, Chrome Frame, Dojo, and more, is often highly critical of JavaScript frameworks. Nonetheless, he approves of Qwik.
+
```

Comment:
Can you mention a specific post about this please?

Reply:
Uhm, how come this is "resolved"?

Reply:
@mhevery asked him directly

Reply:
For posterity: citation added in #5381